pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go

```diff
@@ -225,6 +255,11 @@ func replacePlaceholders(manifest []byte, spec *opv1.OperatorSpec) []byte {
 		pairs = append(pairs, []string{"${LIVENESS_PROBE_IMAGE}", livenessProbe}...)
 	}
 
+	// Cluster ID
+	if clusterID != "" {
+		pairs = append(pairs, []string{"${CLUSTER_ID}", clusterID}...)
```

Comment:
leaving `${CLUSTER_ID}` in there is odd.

Reply:
Removed the condition